AddonExample/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Newtonsoft.Json" version="12.0.3" targetFramework="net45" />
+</packages>
```

Comment:
Can you please migrate this file to `<PackageReference>`s? If you right-click this file in Visual Studio, there should be an option to do that.

Reply:
Changed to PackageReference by 9a8e11d